Assets/Scripts/Source/Game/UI/MenuInput.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+
+public class MenuInput : MonoBehaviour {
+    public EventSystem eventSystem;
```

Comment:
Nit: make `private` + use `[SerializeField]`

Reply:
I am not fond of this.